static/app/views/settings/organizationMembers/organizationMembersList.tsx

```diff
@@ -158,6 +169,30 @@ class OrganizationMembersList extends DeprecatedAsyncView<Props, State> {
     this.setState(state => ({invited: {...state.invited, [id]: 'success'}}));
   };
 
+  handleInviteMissingMember = async (email: string) => {
+    const {organization} = this.props;
+
+    try {
+      const data = await this.api.requestPromise(
+        `/organizations/${organization.slug}/members/`,
+        {
+          method: 'POST',
+          data: {email},
+        }
+      );
+      addSuccessMessage(tct('Sent invite to [email]', {email}));
+      this.setState({members: [...this.state.members, data]});
```

Comment:
Rather than inserting the response into the state, typically its more desirable to re-fetch the full members list

Reply:
should i also do this for updating the missing members list (line 185)?